Formula/stox.rb

```diff
@@ -0,0 +1,20 @@
+class Stox < Formula
+  desc "Swift command-line tool to view and export tickers from finviz.com"
+  homepage "https://github.com/xbladesub/stox"
+  url "https://github.com/xbladesub/stox/archive/1.0.1.tar.gz"
+  sha256 "f037220a0b810775dad6e4df77f335aa9e96ed10b94784ae8d2821a69c628e3b"
+  license "MIT"
+  
+  depends_on xcode: "12.4"
+  
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release", "-Xswiftc", \
+      "-cross-module-optimization", "--enable-test-discovery"
+    mv ".build/release/stox", "stox"
+    bin.install "stox"
+  end
+  
+  test do
+    system "stox", "help", "-n"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Done.